src/web/src/components/Posts/Post.tsx

```diff
@@ -3,6 +3,7 @@ import useSWR from 'swr';
 import { makeStyles, Theme } from '@material-ui/core/styles';
 import { Box, Grid, Typography, ListSubheader, createStyles } from '@material-ui/core';
 import ErrorRoundedIcon from '@material-ui/icons/ErrorRounded';
+import { Container } from 'next/app';
```

Comment:
Looks like this has been deprecated, https://github.com/vercel/next.js/pull/8327.